students/Sally_Shen/lesson4/dict_lab.py

```diff
@@ -0,0 +1,92 @@
+'''
+Dictionaries 1
+Create a dictionary containing “name”, “city”, and “cake” for “Chris” from “Seattle” who likes “Chocolate” (so the keys should be: “name”, etc, and values: “Chris”, etc.)
+Display the dictionary.
+Delete the entry for “cake”.
+Display the dictionary.
+Add an entry for “fruit” with “Mango” and display the dictionary.
+Display the dictionary keys.
+Display the dictionary values.
+Display whether or not “cake” is a key in the dictionary (i.e. False) (now).
+Display whether or not “Mango” is a value in the dictionary (i.e. True).
+'''
+
+dict1 = dict([('name', 'Chris'), ('city', 'Seattle'), ('cake', 'Chocolate')])
+print(dict1)
+
+
+del dict1['cake']
+print(dict1)
+
+dict1['fruit'] = 'Mango'
+print(dict1)
+
+
+print(dict1.keys())
+
+print(dict1.values())
+
+is_key = 'cake' in dict1.keys()
```

Comment:
When looking for a key in a dict, you don't need to specify .keys(). 
Ex: 'cake' in dict1
This will return the same